From 7e2c10d3a39f5347e0f0f41adb29116bfb61ed66 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Sat, 2 Jan 2021 21:48:45 +0900
Subject: [PATCH] Change ruby dir to vendor

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f5db1b..5c2c90b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -545,7 +545,7 @@ if (enable-ruby-interface)
 	message (STATUS " * Ruby:")
 	# find_package(Ruby) found libs of ruby-dev but we only need a directory where we can install ruby libs.
 	find_program (RUBY_EXECUTABLE ruby)
-	EXECUTE_PROCESS(COMMAND ${RUBY_EXECUTABLE} -r rbconfig -e "print RbConfig::CONFIG['rubylibdir']"
+	EXECUTE_PROCESS(COMMAND ${RUBY_EXECUTABLE} -r rbconfig -e "print RbConfig::CONFIG['vendordir']"
 		OUTPUT_VARIABLE RUBY_LIB_DIR)
 	message (STATUS "   Ruby library dir: ${RUBY_LIB_DIR}")
 	if ("${RUBY_LIB_DIR}" STREQUAL "" OR "${RUBY_LIB_DIR}" STREQUAL "nil")
@@ -554,10 +554,6 @@ if (enable-ruby-interface)
 		set (PROGRAMS_MISSING "${PROGRAMS_MISSING} ruby")
 		set (with_ruby no)
 	else()
-		# /usr/lib/ruby/1.8 or /usr/lib/ruby/1.9.1
-		string (REGEX REPLACE "lib/ruby/[0-9].[0-9]?.+" "" RUBY_LIB_DIR_INSTALL "${RUBY_LIB_DIR}")
-		string (REGEX REPLACE "${RUBY_LIB_DIR_INSTALL}" "" RUBY_LIB_DIR_INSTALL "${RUBY_LIB_DIR}")
-		set (RUBY_LIB_DIR "${CMAKE_INSTALL_PREFIX}/${RUBY_LIB_DIR_INSTALL}")
 		message (STATUS "   will be installed in: ${RUBY_LIB_DIR}")
 		set (RUBY_FOUND "TRUE")
 		set (with_ruby yes)
-- 
2.30.0.rc2

