From a3adf34bd20f4bea534fcc1fd03fabada1dbd469 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Sat, 2 Jan 2021 21:58:06 +0900
Subject: [PATCH] Fix support libetpan pkgconfig

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 CMakeLists.txt | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e69a19e..139f7a4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -965,9 +965,8 @@ set (with_mail no)
 # find the compilation flags
 enable_if_not_defined (enable-mail)
 if (enable-mail)
-	find_program (LIBETPAN_CONFIG_EXECUTABLE libetpan-config)
 	execute_process(
-		COMMAND ${LIBETPAN_CONFIG_EXECUTABLE} --cflags
+		COMMAND pkg-config libetpan --cflags
 		OUTPUT_VARIABLE MAIL_PACKAGE_CFLAGS)
 	if (NOT "${MAIL_PACKAGE_CFLAGS}" STREQUAL "" AND NOT "${MAIL_PACKAGE_CFLAGS}" STREQUAL "\n") # if there is a problem with the previous, we don't want to have a lot of errors
 		STRING (REGEX REPLACE "\n" "" TMP_VARIABLE "${MAIL_PACKAGE_CFLAGS}") # to not skip the last option
@@ -980,7 +979,7 @@ if (enable-mail)
 	endif()
 	# find the link flags
 	execute_process(
-		COMMAND ${LIBETPAN_CONFIG_EXECUTABLE} --libs
+		COMMAND pkg-config libetpan --libs
 		OUTPUT_VARIABLE MAIL_PACKAGE_LIBS)
 	if (NOT "${MAIL_PACKAGE_LIBS}" STREQUAL "")
 		STRING (REGEX REPLACE "\n" "" MAIL_PACKAGE_LIBS "${MAIL_PACKAGE_LIBS}")
-- 
2.30.0.rc2

