From 97f4b3baeed5e821f599f60c18ed1d23bbeb964c Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Sun, 3 Jan 2021 01:07:32 +0900
Subject: [PATCH] Swith use libido3-0.1 to libayatana-ido3-0.4

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f5db1b..08e0d9e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -210,7 +210,7 @@ endif()
 ## libido
 enable_if_not_defined (enable-libido-support)
 if (enable-libido-support)
-	set (IDO_MODULE libido3-0.1)
+	set (IDO_MODULE libayatana-ido3-0.4)
 	pkg_check_modules (IDO ${IDO_MODULE})
 	if (IDO_FOUND)
 		STRING (REGEX REPLACE "\\..*" "" IDO_MAJOR "${IDO_VERSION}")
-- 
2.30.0.rc2

