Index: cairo-dock/data/default-theme/launchers/01firefox.desktop
===================================================================
--- cairo-dock.orig/data/default-theme/launchers/01firefox.desktop	2011-02-19 18:23:16.825148184 +0100
+++ cairo-dock/data/default-theme/launchers/01firefox.desktop	2011-02-19 18:23:32.825401475 +0100
@@ -2,12 +2,12 @@
 
 [Desktop Entry]
 #s Launcher's name :
-Name=Firefox Web Browser
+Name=IceWeasel Web Browser
 #s Command to launch on click :
 #{Exemple : nautilus --no-desktop, gedit, etc. You can even enter a keyboard shortcut, for exemple <Alt>F1, <Ctrl>c,  <Ctrl>v, etc}
-Exec=firefox 
+Exec=iceweasel
 #S Image's name or path :
-Icon=firefox
+Icon=iceweasel
 #d Name of the container it belongs to:
 #{If you write here a container's name that doesn't exist yet, a new one will be created, with this icon inside.}
 Container=_MainDock_
@@ -22,7 +22,7 @@
 
 #K Class of the program :
 #{The only reason you may want to modify this parameter is if you made this launcher by hands. If you dropped it into the dock from the menu, it is nearly sure that you shouldn't touch it. It defines the class of the program, which is useful to link the appli with its launcher.}
-StartupWMClass=Firefox
+StartupWMClass=IceWeasel
 
 #b Run in a terminal ?
 #{It will use 'xterm'.}
@@ -41,25 +41,25 @@
 
 Is container=false
 Version=1.0
-Name[ca]=Navegador web Firefox
-Name[cs]=Firefox Webový prohlížeč
-Name[es]=Navegador web Firefox
-Name[fa]=مرورگر اینترنتی Firefox
-Name[fi]=Firefox-selain
-Name[fr]=Firefox Web Browser
-Name[hu]=Firefox webböngésző
-Name[it]=Firefox Browser Web
-Name[ja]=Firefox ウェブ・ブラウザ
-Name[ko]=Firefox 웹 브라우저
-Name[nb]=Firefox Nettleser
-Name[nl]=Firefox webbrowser
-Name[nn]=Firefox Nettlesar
-Name[no]=Firefox Nettleser
-Name[pl]=Przeglądarka WWW Firefox
-Name[pt]=Firefox Navegador Web
-Name[pt_BR]=Navegador Web Firefox
-Name[sk]=Internetový prehliadač Firefox
-Name[sv]=Webbläsaren Firefox
+Name[ca]=Navegador web IceWeasel
+Name[cs]=IceWeasel Webový prohlížeč
+Name[es]=Navegador web IceWeasel
+Name[fa]=مرورگر اینترنتی IceWeasel
+Name[fi]=IceWeasel-selain
+Name[fr]=IceWeasel Web Browser
+Name[hu]=IceWeasel webböngésző
+Name[it]=IceWeasel Browser Web
+Name[ja]=IceWeasel ウェブ・ブラウザ
+Name[ko]=IceWeasel 웹 브라우저
+Name[nb]=IceWeasel Nettleser
+Name[nl]=IceWeasel webbrowser
+Name[nn]=IceWeasel Nettlesar
+Name[no]=IceWeasel Nettleser
+Name[pl]=Przeglądarka WWW IceWeasel
+Name[pt]=IceWeasel Navegador Web
+Name[pt_BR]=Navegador Web IceWeasel
+Name[sk]=Internetový prehliadač IceWeasel
+Name[sv]=Webbläsaren IceWeasel
 Comment=Browse the World Wide Web
 Comment[ca]=Navegueu per el web
 Comment[cs]=Prohlížení stránek World Wide Webu
