From f3ce31c5e1472015a6b4102f725697a3c9d4ec22 Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@debian.org>
Date: Fri, 25 May 2018 18:25:37 +0900
Subject: [PATCH] Use long ID of GPG

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@debian.org>
---
 data/scripts/help_scripts.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/scripts/help_scripts.sh b/data/scripts/help_scripts.sh
index e699761..893fe5e 100755
--- a/data/scripts/help_scripts.sh
+++ b/data/scripts/help_scripts.sh
@@ -58,13 +58,13 @@ repository() {
 
 ppa() {
 	addRepo "deb http://ppa.launchpad.net/cairo-dock-team/ppa/ubuntu $(lsb_release -sc) main" "Cairo-Dock-PPA"
-	apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E80D6BF5
+	apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4E6B5E978E5D9070EBC365BA7274A4DAE80D6BF5
 	up_install
 }
 
 weekly() {
 	addRepo "deb http://ppa.launchpad.net/cairo-dock-team/weekly/ubuntu $(lsb_release -sc) main" "Cairo-Dock-PPA-Weekly"
-	apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E80D6BF5
+	apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 4E6B5E978E5D9070EBC365BA7274A4DAE80D6BF5
 	up_install
 }
 
-- 
2.30.0.rc2

